downstream/modules/aap-hardening/ref-aap-authentication.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies: 
+// downstream/assemblies/assembly-hardening-aap.adoc
+
+[id="ref-aap-authentication_{context}"]
+
+= {PlatformNameStart} authentication
+
+[role="_abstract"]
+
+{ControllerNameStart} currently supports the following external authentication mechanisms through the {GatewayStart} UI:
+
+* Azure Activity Directory
```

Comment:
this bullet list should be rewritten to match the list here: https://docs.redhat.com/en/documentation/red_hat_ansible_automation_platform/2.5/html/access_management_and_authentication/gw-configure-authentication#gw-config-authentication-type